README.md

```diff
@@ -67,9 +67,7 @@ all. In that case, you can put the `bash_completion` file in `/sw/etc`
 and add the following code to `~/.bash_profile`:
 
 ```shell
-if [ -f /sw/etc/bash_completion ]; then
-   . /sw/etc/bash_completion
-fi
+[[ -r "$HOMEBREW_PREFIX/etc/profile.d/bash_completion.sh" ]] && . "$HOMEBREW_PREFIX/etc/profile.d/bash_completion.sh"
```

Comment:
- `if ... then ... fi` should be used. `[[ ... ]] && . ...` will leave `$?` being `1` for the first prompt when the file is not found.
- If you want to specifically check the read attribute by `-r` (instead of using `-f`), you may be further specific to request a non-empty file by `-s`.

Reply:
Added - whoever is maintaining the Homebrew site should update accordingly